src/widgets/call-panel.jsx

```diff
@@ -139,6 +149,16 @@ class CallPanel extends React.PureComponent {
     }
   }
 
+  // Creates an empty video track placeholder.
+  emptyVideoTrack() {
+    const width = 640;
+    const height = 480;
+    let canvas = Object.assign(document.createElement("canvas"), {width, height});
+    canvas.getContext('2d').fillRect(0, 0, width, height);
+    let stream = canvas.captureStream();
```

Comment:
const stream ...
You should probably use explicit frame rate of 0, i.e.
`const stream = canvas.captureStream(0);`

Reply:
Done.